### DISCUSSION TITLE
邮箱自动补全？

### DISCUSSION THREAD

**Discussion by gclintj** [rime/home#771] [Ideas]
比如在中文输入法下输入一些字母后加@会自动建议常用email域名补全，并记录曾经输入过的邮箱以便以后再次输入。
好像到处搜了一下还没有类似功能。

随便，有类似的功能建议，此Discussions是最好的途径吗？

另外，感觉想使用这个输入法的学习曲线很陡峭，除了读技术文档，还有别的推荐学习使用的路线吗？

**Comment by lotem** [comment]
在這兒討論正好。

我主張稱其為輸入法定制框架。

可以利用現有機制部分實現該需求，如仿照此例
https://github.com/rime/home/wiki/CustomizationGuide#%E6%B4%BB%E7%94%A8%E6%A8%99%E9%BB%9E%E5%89%B5%E5%BB%BA%E8%87%AA%E5%AE%9A%E7%BE%A9%E8%A9%9E%E7%B5%84
改`@`符號為多選，提示出域名後綴。

如果要再加上逐鍵補全、記錄的功能就得另外開發了，還需要有數據支持。
